cfme/intelligence/reports/reports.py

```diff
@@ -225,6 +226,24 @@ def is_displayed(self):
         )
 
 
+class ReportScheduleView(SchedulesFormCommon):
```

Comment:
Hang on I'm pretty sure all of this already exists in the schedules.py module:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/intelligence/reports/schedules.py